ivy_tests/array_api_methods_to_test/elementwise_functions.txt

```diff
@@ -53,4 +53,4 @@ sqrt
 subtract
 tan
 tanh
-trunc
+trunc
```

Comment:
Please update this file so that it doesn't appear in the files changed, as we shouldn't be making changes to the array api test suite at all

Reply:
@VedPatwardhan I have not changed anything in it. It is same as [master](https://github.com/unifyai/ivy/blob/master/ivy_tests/array_api_methods_to_test/elementwise_functions.txt). Also when I try to remove it, [it shows same thing for the previous line](https://github.com/unifyai/ivy/pull/4811/commits/096885d09a348427d491b8f0ae1cb9527492f2ad#diff-1b93f37da8c5d9bd34902cecc90d492dab74f69db1afc6b04c68da7517443b3f).